codemods/mocha/vitest/migrate-tests/src/index.ts

```diff
@@ -40,6 +48,32 @@ export default function transform(
 		},
 	});
 
+	root.find(j.ImportDeclaration).forEach((importDeclaration) => {
+		importDeclaration.node.specifiers?.forEach(
+			(
+				specifier:
+					| ImportSpecifier
```

Comment:
why did you need to declare types?

Reply:
@hbjORbj the imports are still imported, linting has failed

Reply:
addressed